pkg/resource/alloy-secret/alloy-logging-secret.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
```

Comment:
Is thé app not expecting clusterId-logging-secret? Why thé need to add alloy?

Reply:
Yes the final format is `$clusterID-alloy-logging-secret`. I added the `logging` part to make it explicit that this secret is intended for logging usage.